frontend/src/model/beta/releases/ModelReleaseDisplay.tsx

```diff
@@ -105,33 +105,41 @@ export default function ModelReleaseDisplay({
               </Typography>
             </Typography>
             <Markdown>{release.notes}</Markdown>
-            {(release.files.length > 0 || release.images.length > 0) && <Divider />}
-            <Stack>
-              {release.files.map((file) => (
-                <Stack
-                  key={file._id}
-                  direction={{ sm: 'row', xs: 'column' }}
-                  justifyContent='space-between'
-                  alignItems='center'
-                  spacing={2}
-                >
-                  <Link href='/beta'>{file.name}</Link>
-                  <Typography variant='caption'>{prettyBytes(file.size)}</Typography>
-                </Stack>
-              ))}
-              {release.images.map((image) => (
-                <Stack
-                  key={image}
-                  direction={{ sm: 'row', xs: 'column' }}
-                  justifyContent='space-between'
-                  alignItems='center'
-                  spacing={2}
-                >
-                  <Link href='/beta'>{image}</Link>
-                  {/* TODO - Add file size here */}
-                  {/* <Typography variant='caption'>123GB</Typography> */}
-                </Stack>
-              ))}
+            <Box>{(release.files.length > 0 || release.images.length > 0) && <Divider />}</Box>
+            <Stack spacing={2}>
+              {release.files.length > 0 && (
+                <Box>
+                  <Typography fontWeight='bold'>Artefacts</Typography>
+                  {release.files.map((file) => (
+                    <Stack
+                      key={file._id}
+                      direction={{ sm: 'row', xs: 'column' }}
+                      justifyContent='space-between'
+                      alignItems='center'
+                      spacing={2}
+                    >
+                      <Link href='/beta'>{file.name}</Link>
+                      <Typography variant='caption'>{prettyBytes(file.size)}</Typography>
+                    </Stack>
+                  ))}{' '}
+                </Box>
+              )}
+              {release.images.length > 0 && (
+                <Box>
+                  <Typography fontWeight='bold'>Docker images</Typography>
+                  {release.images.map((image) => (
+                    <Stack
+                      key={`${image.model}-${image.version}`}
+                      direction={{ sm: 'row', xs: 'column' }}
+                      justifyContent='space-between'
+                      alignItems='center'
+                      spacing={2}
+                    >
+                      <Link href='/beta'>{`${image.model}-${image.version}`}</Link>
```

Comment:
This should probably not link anywhere, but instead when clicked 'copy' the text.  It should also be of the format:

```
publicaddress/modelId/name:tag
```

So that people can just paste it in to `docker pull`.